colabs/prompts/W&B_LLM_QA_Bot.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    from wandb.integration.langchain import WandbTracer
</pre><p>This is deprecated and we should only use the environment variable or load this directly from langchain and not our SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>